catalogd/testdata/catalogs/test-catalog.Dockerfile

```diff
@@ -1,6 +0,0 @@
-FROM scratch
```

Comment:
@tmshort I saw that you removed from opc/release already :-) 
So, it will not broke the bumper